### ISSUE TITLE
maintained plugin?

### ISSUE THREAD

**Issue by bwp91** [SANdood/homebridge-purpleair#8]
Hi @SANdood and @beniaminrychter - tagging you both as you both are listed as contributors to this repo 😁

I'm from the homebridge team and just wondered if this plugin is still maintained?

If it is not, would you consider transferring this repo (and the npm package) to the homebridge unmaintained repo?
https://github.com/homebridge/unmaintained-plugins

It basically means that someone else can continue the hard work under the `homebridge-purpleair` name at some point if they wish!

Cheers
Ben

**Comment by tillkruss** [comment]
Replacement: https://github.com/tillkruss/homebridge-purpleair

**Comment by SANdood** [comment]
Yes - as far as I am concerned, feel free to transfer the package to your unmaintained plugins...or tell me how I make that transfer...

Thanks!
	Barry

**Comment by tillkruss** [comment]
@SANdood if you log into npmjs.com and make me the owner of the package, then I can take it over 👍

My username is `tillkruss`.

**Comment by bwp91** [comment]
@SANdood if you are happy to continue, please:

1. transfer the github repo to the homebridge plugins repo (you can do this via repo settings, and transfer in the 'danger zone at the bottom' - I haven't done this recently so if it looks _safer_ to transfer it to me then also happy with this, my username is `bwp91`
2. add me as a collaborator on the npm repo via the settings page on the package page. again, my username here is `bwp91`

**Comment by SANdood** [comment]
Done.

**Comment by bwp91** [comment]
I can see number (1) is complete now, but I don't see number (2). Did you add me as a collaborator to the package via the npmjs.com website?

Cheers

**Comment by SANdood** [comment]
It says that it sent you an invitation:

￼

**Comment by bwp91** [comment]
sorry, yes, email had gone to spam 👍🏻
